pkg/utils/file.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	kopsutils "k8s.io/kops/upup/pkg/fi/utils"
+	"os"
+)
+
+// FileExists checks to see if a file exists.
+func FileExists(path string) (bool, error) {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+
+	return false, err
+}
+
+// CheckFileExists returns an error if the file doesn't exist
```

Comment:
Would returning a boolean be insufficient?

Reply:
Yeah, I just returned an error because that's usually the convention with the "checkSomething()" methods but the boolean is nicer to use anyway. Will change